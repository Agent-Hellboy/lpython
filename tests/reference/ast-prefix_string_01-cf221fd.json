{
    "basename": "ast-prefix_string_01-cf221fd",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/errors/prefix_string_01.py",
    "infile_hash": "0d83c0e32a78023fccb343a4d3358071792265c1ae357176fe0912eb",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "ast-prefix_string_01-cf221fd.stderr",
    "stderr_hash": "41f6e79aaf62e7c95aeaa048a93b68887576db7d7d80f755073e9665",
    "returncode": 1
}