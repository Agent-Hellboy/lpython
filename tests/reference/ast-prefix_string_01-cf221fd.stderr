Internal Compiler Error: Unhandled exception
Traceback (most recent call last):
[2m  File "[0m[1m[35m$DIR/src/bin/lpython.cpp[39m[0m[2m", line 1775[0m
    if (show_tokens) {
[2m  File "[0m[1m[35m$DIR/src/bin/lpython.cpp[39m[0m[2m", line 154[0m
    compiler_options.use_colors) << std::endl;
[2m  File "[0m[1m[35m$DIR/src/lpython/parser/parser.cpp[39m[0m[2m", line 126[0m
    Result<LPython::AST::Module_t*> res = parse(al, input, prev_loc, diagnostics);
[2m  File "[0m[1m[35m$DIR/src/lpython/parser/parser.cpp[39m[0m[2m", line 22[0m
    p.parse(s, prev_loc);
[2m  File "[0m[1m[35m$DIR/src/lpython/parser/parser.cpp[39m[0m[2m", line 53[0m
    m_tokenizer.set_string(inp, prev_loc);
[2m  File "[0m[1m[35mparser.yy[39m[0m[2m", line 1107[0m
[2m  File "[0m[1m[35m$DIR/src/lpython/parser/semantics.h[39m[0m[2m", line 879[0m
    throw LCompilers::LCompilersException("The string is not recognized by the parser.");
LCompilersException: The string is not recognized by the parser.
